code/game/mecha/mech_fabricator.dm

```diff
@@ -62,19 +62,19 @@
 		// FULPSTATION: Tiered part sets
 
 		part_sets = list(
-								"Cyborg","Misc"
+								"Cyborg", "Cyborg Upgrade Modules", "Misc"
 								)
 		if( Ml.rating >= 2)
 			part_sets += list(
-								"Ripley","Exosuit Equipment","Exosuit Ammunition","Cyborg Upgrade Modules"
+								"Exosuit Equipment", "Clarke", "Ripley"

```

Comment:
almost knee-jerk reacted to the firefighter replacement being alongside the ripley, but iirc its a sidegrade instead of a straight upgrade now so this is fine